src/gui/guiFormSpecMenu.cpp

```diff
@@ -3265,7 +3265,7 @@ void GUIFormSpecMenu::regenerateGui(v2u32 screensize)
 						((15.0 / 13.0) * (0.85 + mydata.invsize.Y));
 			}
 
-#ifdef __ANDROID__
+#ifdef HAVE_TOUCHSCREENGUI
```

Comment:
also an android/small screen thing

Reply:
It seems to be more of a smaller screen size thing from the comment, which would make sense on a touchscreen phone. Maybe this should a a 3rd independent parameter?

Reply:
Yeah that'd be ideal but this is not a change that should be done in this PR.
You can keep it like this for now.